content/docs/command-reference/config.md

```diff
@@ -9,7 +9,7 @@ usage: dvc config [-h] [--global | --system | --local] [-q | -v] [-u]
                   [-l] [name] [value]
 
 positional arguments:
-  name           Option name.
+  name           Option name (remote.name.option or section.option).
```

Comment:
Have thought about using `[remote.]<section>.<option>` in metavar, but seems to be a bit too long for that.

Reply:
Maybe we should put `section.option` first?

Reply:
Does this affect `remote modify` at all? Or `cache dir`

Reply:
Good point about the ordering. Will adjust. Thanks! :pray: 

> Does this affect remote modify at all? Or cache dir

Doesn't affect any other commands, this is purely `dvc config` thing.